aospy/automate.py

```diff
@@ -295,19 +295,28 @@ def _exec_calcs(calcs, parallelize=False, client=None, **compute_kwargs):
         def func(calc):
             """Wrap _compute_or_skip_on_error to require only the calc
             argument"""
-            return _compute_or_skip_on_error(calc, compute_kwargs)
+            return _compute_or_skip_on_error(calc, {'write_to_tar': False})
```

Comment:
This causes none of the other compute_kwargs to be passed.  Maybe update the value instead?

```python
compute_kwargs.update({'write_to_tar': False})
_compute_or_skip_on_error(calc, compute_kwargs)
```

Reply:
(I recognize that currently 'write_to_tar' is the only supported kwarg, so for now this is irrelevant.  But this leaves the door open for other options in the future.)